.github/workflows/merge-flow.yaml

```diff
@@ -77,9 +77,28 @@ jobs:
           echo "upstream=$(dirname ${{ inputs.upstream }})" >> $GITHUB_OUTPUT
           echo "downstream=$(dirname ${{ inputs.downstream }})" >> $GITHUB_OUTPUT
           echo "sandbox=$(dirname ${{ inputs.sandbox }})" >> $GITHUB_OUTPUT
+          # assumes the presence of ./VERSION
+          DOWNSTREAM_VERSION=$(curl -sL "https://raw.githubusercontent.com/${{ inputs.downstream }}/${{ inputs.downstream-branch }}/VERSION")
+          echo "downstream-version=${DOWNSTREAM_VERSION}" >> $GITHUB_OUTPUT
+      - uses: madhead/semver-utils@latest
+        id: version
+        with:
```

Comment:
could we set `lenient: false` to ensure that the action fails in case it can't parse the version?